packages/backend/src/helpers/send-email.ts

```diff
@@ -5,26 +5,28 @@ import HttpError from '@/errors/http'
 
 import logger from './logger'
 
-export async function sendEmail({
-  subject,
-  body,
-  recipient,
-}: {
+export interface PostmanEmailRequestBody {
   subject: string
   body: string
   recipient: string
-}): Promise<void> {
+  replyTo?: string
+}
+
+export async function sendEmail(
+  request: PostmanEmailRequestBody,
```

Comment:
actually u can destructure the keys here 
e.g.
{ subject, body, ... }: PostmanEmailRequestBody

Reply:
hmm okok ic i thought this looks neater oh welps